templates/contrib/facets/facets-item-list--checkbox.html.twig

```diff
@@ -133,7 +133,7 @@
               <div class="checkbox-accordion">
                 <div class="accordion--content" aria-hidden="true" hidden="hidden" id="hidden-options-more-than-6-">
                   <div{{ attributes.addClass('links checkbox-filter__checkboxes') }}>
-                    {%- for item in items|slice(4, 6) -%}
+                    {%- for item in items|slice(3, 6) -%}
```

Comment:
@lennartvava From 6 to 20 checkboxes: The first 3 checkboxes are shown. The following checkboxes are in an accordion with a “Show more” button to show or to hide them.
```suggestion
                    {%- for item in items|slice(3, 19) -%}
```